src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -139,7 +139,8 @@ public PodTemplateBuilder(PodTemplate template, KubernetesSlave agent) {
 
     private static String getDefaultImageName() {
       // TODO: Reverse logic after inbound-agent:4.9-1
-      String name = "jenkins/inbound-agent:4.3-4";
+      String name = "jenkins/inbound-agent:latest";
```

Comment:
The TODO line just above should be addressed.

Reply:
I'll made an updatecli manifest to keep this value up to date on Monday if it's OK for everyone :)

Reply:
updatecli is better than nothing, though Dependabot generally works better when it is feasible since it has more sophisticated behavior and better integration into other GH features.

Reply:
That works for me, whatever gets a newer version in there that isn't 2+ years old 🙂

Reply:
Implemented in https://github.com/jenkinsci/kubernetes-plugin/pull/1085